clang/lib/Driver/ToolChains/Gnu.cpp

```diff
@@ -472,6 +472,22 @@ void tools::gnutools::Linker::ConstructJob(Compilation &C, const JobAction &JA,
   // The profile runtime also needs access to system libraries.
   getToolChain().addProfileRTLibs(Args, CmdArgs);
 
```

Comment:
Nope.  I left it that way in case something went wrong when I was testing and I needed to add things back in and simply forgot to delete it after verifying all was well. I have removed the commented section.